<?xml version="1.0"?>
<launch>

<!-- Play rosbag -->
<arg name="bag" default="$(find reference_state_generator)/data/poseimu.bag"/>
<param name="use_sim_time" value="true"/>
<node pkg="rosbag" type="play" name="rosbag_play" args="-r 1 --clock $(arg bag)" required="true" /> 

<!-- Launch rosario gtsam node -->
  <node pkg="reference_state_generator" type="ref_state_generator.py" name="ref_state_generator" output="screen">
   <!-- <rosparam command="load" file="$(find rosario_gtsam)/config/rosario_params.yaml" /> -->
  </node>

<node pkg="tf" type="static_transform_publisher" name="link1_broadcaster" args="0 0 0 0 0 0 1 map global 50" />


<!-- Launch rviz -->
<node type="rviz" name="rviz" pkg="rviz" args="-d $(find reference_state_generator)/launch/show_pose.rviz" />

<!-- GUI for Testing  -->
  <!-- <node name = "rqt_gui" pkg = "rqt_gui" type = "rqt_gui" respawn = "false" output = "screen" 
  args = "-perspective-file $(find rosario_gtsam)/rqt_multiplot/rqt_multiplot.perspective"/> -->



</launch>

<!-- launch-prefix="gdb -ex run -args" -->       
